frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/model/AttributeValues.java

```diff
@@ -16,28 +16,51 @@
 
 package nl.nn.adapterframework.frankdoc.model;
 
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 import lombok.Getter;
+import nl.nn.adapterframework.frankdoc.doclet.FrankDocException;
 
 public class AttributeValues {
 	private @Getter String fullName;
 	private String simpleName;
-	private @Getter List<String> values;
+	private final @Getter List<String> values;
+	private final Set<String> valueSet;
 	private int seq;
 
 	AttributeValues(String fullName, String simpleName, List<String> values, int seq) {
 		this.fullName = fullName;
 		this.values = values;
+		this.valueSet = getValueSet(values);
 		this.simpleName = simpleName;
 		this.seq = seq;
 	}
 
+	private static Set<String> getValueSet(List<String> values) {
+		// Value null does not make sense, but allowing it is useful for unit tests
+		// that are not concerned with the values.
+		if(values == null) {
+			return new HashSet<>();
```

Comment:
I can remove this if by fixing my unit tests. A null is only used in unit tests about AttributeValues. If the unit tests construct empty AttributeValue instances with an empty list instead of null, then I do not have to support null values here. Shall I fix this?

Reply:
Gedaan.